test/e2e/DCAHub/happy-path.spec.ts

```diff
@@ -254,7 +254,7 @@ contract('DCAHub', () => {
       setSwapRatio(swapRatioBC2);
       await evm.advanceTimeAndBlock(SwapInterval.ONE_HOUR.seconds);
 
-      await assertIntervalsToSwapNowAre(SwapInterval.FIFTEEN_MINUTES, SwapInterval.ONE_HOUR);
```

Comment:
Now that the only position with 1h intervals has ended, it will no longer be present